bowler/main.py

```diff
@@ -138,5 +138,24 @@ def run(codemod: str, argv: List[str]) -> None:
         sys.argv[1:] = original_argv
 
 
+@main.command()
+@click.argument("codemod", required=True, type=str)
+def test(codemod: str) -> None:
+    """
+    Run the tests in the codemod file 
+        """
+    import unittest
+    import importlib.util
+    import os.path
+
+    module_name_from_codemod = os.path.basename(codemod).replace(".py", "")
+    spec = importlib.util.spec_from_file_location(module_name_from_codemod, codemod)
+    foo = importlib.util.module_from_spec(spec)
+    spec.loader.exec_module(foo)
+    suite = unittest.TestLoader().loadTestsFromModule(foo)
+
+    unittest.TextTestRunner().run(suite)
```

Comment:
Can you make sure this exits nonzero on failure?  Looking at what unittest.main does is probably useful here.